src/connector/src/source/cdc/source/reader.rs

```diff
@@ -119,38 +119,56 @@ impl CommonSplitReader for CdcSplitReader {
             properties.insert("table.name".into(), table_name);
         }
 
-        let cdc_stream = cdc_client
-            .start_source_stream(
-                self.source_id,
-                self.conn_props.get_source_type_pb()?,
-                self.start_offset,
-                properties,
-                self.snapshot_done,
-            )
-            .await
-            .inspect_err(|err| tracing::error!("connector node start stream error: {}", err))?;
-        pin_mut!(cdc_stream);
-        #[for_await]
-        for event_res in cdc_stream {
-            match event_res {
-                Ok(GetEventStreamResponse { events, .. }) => {
-                    if events.is_empty() {
-                        continue;
-                    }
-                    let mut msgs = Vec::with_capacity(events.len());
-                    for event in events {
-                        msgs.push(SourceMessage::from(event));
-                    }
-                    yield msgs;
-                }
-                Err(e) => {
-                    return Err(anyhow!(
-                        "Cdc service error: code {}, msg {}",
-                        e.code(),
-                        e.message()
-                    ))
-                }
+        let (tx, mut rx) = mpsc::channel(1024);
+        let tx: Box<GetEventStreamJniSender> = Box::new(tx);
+
+        JVM.as_ref()?;
```

Comment:
may remove it?

Reply:
This one is necessary because we need to ensure it is initialized in the following thread.

Reply:
We may use `LazyLock::force` to make it more explicit. :)